system/jlib/jutil.cpp

```diff
@@ -79,6 +81,10 @@ MODULE_INIT(INIT_PRIORITY_SYSTEM)
     if (mach_timebase_info(&timebase_info) != KERN_SUCCESS)
         return false;
 #endif
+    hpccBuildTag = getenv("HPCC_BUILD_TAG");
+    if (isEmptyString(hpccBuildTag))
+        hpccBuildTag = hpccBuildTag;
```

Comment:
I suspect this should say

hpccBuildTag = BUILD_TAG;

Reply:
whoops, yes it did, a s&r gone wrong.